auth0/src/main/java/com/auth0/android/result/SessionTransferCredentials.kt

```diff
@@ -37,18 +44,22 @@ public data class SSOCredentials(
     @field:SerializedName("token_type") public val tokenType: String,
 
     /**
-     * Expiration duration of the session token in seconds. Session tokens are short-lived and expire after a few minutes.
-     * Once expired, the Session Token can no longer be used for SSO.
+     * Expiration duration of the web sso token in seconds. Web SSO tokens are short-lived and expire after a few minutes.
+     * Once expired, the web sso token can no longer be used for SSO.
      *
-     * @return the expiration duration of this Session Token
+     * @return the expiration duration of this web sso token
      */
     @field:SerializedName("expires_in") public val expiresIn: Int,
 
-
     /**
      *  Refresh Token that can be used to request new tokens without signing in again.
      *
      * @return the Refresh Token.
      */
     @field:SerializedName("refresh_token") public val refreshToken: String? = null
-)
\ No newline at end of file
+) {
+
+    override fun toString(): String {
+        return "SSOCredentials(sessionTransferToken = ****, issuedTokenType = $issuedTokenType, tokenType = $tokenType, expiresIn = $expiresIn, refreshToken = ****)"
```

Comment:
This still has the old `SSOCredentials` name. And it's not including the ID token (which should be redacted).